src/core/log.js

```diff
@@ -98,12 +98,3 @@ pc.extend(pc, function () {
         log: log
     };
 }());
-
-// Shortcuts to logging functions
-var logINFO = pc.log.info;
-var logDEBUG = pc.log.debug;
-var logWARNING = pc.log.warning;
-var logERROR = pc.log.error;
-
-var logALERT = pc.log.alert;
-var logASSERT = pc.log.assert;
```

Comment:
no one seems to use these anywhere.

Reply:
These are there for backwards compatibility purposes.

Reply:
Hmm, how can anyone access them from outside?

Reply:
They're defined in global scope.

Reply:
Ah I see, because of the way you guys build, all js files are concatenated directly.  Sorry, I was unaccustomed to that style.  I am used to those being module local variables.